README.md

```diff
@@ -1,8 +1,33 @@
 
-# advance-valuation-rulings
+# Advance Valuation Ruling Backend
 
-This is a placeholder README.md for a new repository
+Service providing the public frontend to ARS.
 
-### License
+## Frontend
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+[Advance Valuation Ruling](https://github.com/hmrc/advance-valuation-rulings-frontend)
```

Comment:
I suspect this would be something like:
`[Advance Valuation Rulings Frontend](https://github.com/hmrc/advance-valuation-rulings-frontend)`